### DISCUSSION TITLE
Installing updates

### DISCUSSION THREAD

**Discussion by mbutii** [KRTirtho/spotube#781] [Q&A]
Hello,
I have 3.1.1 installed, which I installed from the app manager of Linux Mint. Spotube tells me a new version (3.1.2) and if I click the update button it brings me to a [webpage](https://spotube.krtirtho.dev/other-downloads/stable-downloads) for downloading the new version manually.
Is there a way to update from Linux Mint's App Manager instead of manually, so that it's easier and I don't do crap?

**Comment by KRTirtho** [comment]
I assume by App Manager u mean Software center that installs the Flatpak version of Spotube. Spotube is already at v3.1.2 in Flatpak, so I don't see any reason for not being able to upgrade

**Reply by KRTirtho** [reply]
Just upgrade the package from App Manager like any other app
